### ISSUE TITLE
Toggl track browser extension not populating Tags and Project inputs in entry dialog.

### ISSUE THREAD

**Issue by thezwap** [toggl/track-extension#2298]
* **OS version**: 
 Windows 11
* **Browser version**: 
Brave Version 1.64.122 Chromium: 123.0.6312.122 (Official Build)  (64-bit)
* **Extension version**: 
3.0.32

#### Relevant integration (if any):

Hubspot (app.hubspot.com)

### 🐛 Describe the bug

When starting the timer, the tags and project are not populated.

### Expected behaviour

If I have valid CSS selectors, then the Project and Tags inputs should be populated correctly.

### Steps to reproduce

1. Go to a Hubspot ticket
2. Click "Start Timer"
3. Note that Project and Tags are empty

### Other details or context

* I've tried hardcoding values for project and tags in `togglbutton.createTimerLink()` in the integration code, to eliminate the CSS selectors from the equation. This did not work
* After debugging index.js (for the extension) I noticed the `tag_ids` field and added that to the object passed to `togglbutton.createTimerLink()` in the extension code, and that didn't work either.
* While debugging index,js, I can see the values for `tags` and `tag_ids` but it's hard to follow the minified code. They must be getting dropped or ignored somewhere along the way, before the toggl entry dialog opens.

**Comment by thezwap** [comment]
Anybody?

**Comment by willrowe** [comment]
I'm seeing this as well.
